deployment/terraform/anonymize_db_job.tf

```diff
@@ -0,0 +1,155 @@
+# https://registry.terraform.io/modules/cn-terraform/ecs-fargate-task-definition/aws/latest
+module "database_anonymizer_task_definition" {
+  source                 = "git@github.com:cn-terraform/terraform-aws-ecs-fargate-task-definition.git?ref=1.0.36"
+  name_prefix            = "database-anonymizer"
+  container_image        = local.database_anonymizer_image
+  container_name         = "database-anonymizer"
+  #ephemeral_storage_size = 25
+
+  environment = [
+    {
+      name: "POSTGRES_USER",
+      value: "anondb"
+    },
+    {
+      name: "POSTGRES_PASSWORD",
+      value: "anondb"
+    },
+    {
+      name: "POSTGRES_DB",
+      value: "anon"
+    },
+    {
+      name: "SOURCE_POSTGRES_HOST",
+      value: aws_route53_record.database.name
+    },
+    {
+      name: "SOURCE_POSTGRES_PORT",
+      value: module.database_enc.port
+    },
+    {
+      name: "SOURCE_POSTGRES_USER",
+      value: var.rds_database_username
+    },
+    {
+      name: "SOURCE_POSTGRES_PASSWORD",
+      value: var.rds_database_password
+    },
+    {
+      name: "SOURCE_POSTGRES_DB",
+      value: var.rds_database_name
+    },
+    {
+      name: "AWS_DEFAULT_REGION",
+      value: var.aws_region
+    },
+    {
+      name: "AWS_STORAGE_BUCKET_NAME",
+      value: var.anonymized_db_s3_storage
+    }
+  ]
+
+#  secrets = [
+#    {
+#       valueFrom: "arn:aws:ssm:eu-west-1:343975343274:parameter/test",
+#       name: "TEST"
+#    },
+#  ]
+
+  log_configuration = {
+    logDriver: "awslogs",
+    options: {
+      "awslogs-group": aws_cloudwatch_log_group.database_anonymizer.name,
+      "awslogs-region": var.aws_region,
+      "awslogs-stream-prefix": "database-anonymizer"
+    }
+  }
+}
+
+# AWS ECS Fargate Schedule Task Terraform Module
+# https://registry.terraform.io/modules/cn-terraform/ecs-fargate-scheduled-task/aws/latest
+module "database_anonymizer_task" {
+  source                                      = "git::git@github.com:cn-terraform/terraform-aws-ecs-fargate-scheduled-task.git?ref=1.0.25"
+  name_prefix                                 = "anonymize_database-task"
+  event_rule_name                             = "anonymize_database-rule"
+  event_rule_schedule_expression              = "cron(50 15 * * ? *)"
+  ecs_cluster_arn                             = module.database_anonymizer_cluster.aws_ecs_cluster_cluster_arn
+  event_target_ecs_target_subnets             = module.vpc.public_subnet_ids
+  event_target_ecs_target_task_definition_arn = module.database_anonymizer_task_definition.aws_ecs_task_definition_td_arn
+  ecs_execution_task_role_arn                 = aws_iam_role.database_anonymizer.arn
+  ecs_task_role_arn                           = aws_iam_role.database_anonymizer.arn
+}
+
+module "ecr_repository_database_anonymizer" {
+  source = "github.com/azavea/terraform-aws-ecr-repository?ref=1.0.0"
+
+  repository_name =  "${lower(replace(var.project, " ", ""))}-database-anonymizer-${lower(var.environment)}"
```

Comment:
You can set these constructions in files like local.tf and then use local.vars here in a simpler view